docs/UPGRADING.md

```diff
@@ -0,0 +1,9 @@
+# Upgrading
+
+Here you can find a list of migration guides to handle breaking changes between releases of the CLI.
+
+For more information about breaking changes see the [contributing documentation][contrib-docs].
```

Comment:
I don't think the linked information is of interest to the target reader of this file, so I would just remove this sentence.

If it is to be retained, please change the link to be relative:
```markdown
[contrib-docs]: CONTRIBUTING.md#breaking
```

Reply:
Right, didn't think about that. I'll just remove it.